chain.go

```diff
@@ -154,13 +154,6 @@ type Chain[B Block] struct {
 }
 
 type ToolsConfig[B Block] struct {
```

Comment:
why did we lose the sanitization option ? I feel that it may have been useful in some cases

Reply:
I need it for Ethereum compare because `failureReason` are different so we exclude them as part of the compare.

Reply:
ok I added it back with a new format that will work. minimal adjustments in fireeth will be needed.